images/saige-qtl/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM australia-southeast1-docker.pkg.dev/analysis-runner/images/driver-r:1.4
+
+ENV MAMBA_ROOT_PREFIX /root/micromamba
+
+SHELL ["/bin/bash", "-eo", "pipefail", "-c"]
+RUN micromamba install -y --prefix $MAMBA_ROOT_PREFIX \
+        -c cpg -c bioconda -c conda-forge \
+        r-essentials r-base=4.1.2 python=3.10 \
+        anaconda cmake \
+        gettext lapack r-matrix \
+        r r-rcpp r-rcpparmadillo r-data.table r-bh r-matrix \
+        r-spatest r-rcppeigen r-devtools r-skat \
+        r-rcppparallel r-optparse boost openblas r-rhpcblasctl r-metaskat \
+        r-skat r-qlcmatrix r-rsqlite \
+    rm -r /root/micromamba/pkgs \
+    pip3 install cget click
```

Comment:
Did you mean to concatenate these with `&&` at the end of the lines?

Reply:
yes sorry my bad